### PR TITLE
Mention documentation in Python logging README

### PR DESCRIPTION
**Pull request by geigerj**

### CODE REVIEW

File: generated/python/gapic-google-cloud-logging-v2/setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='gapic-google-cloud-logging-v2',
-    version='0.14.0',
+    version='0.14.1',
```
Comment (ethanbao) [inline generated/python/gapic-google-cloud-logging-v2/setup.py]: Why do you include this change in this PR?

Comment (geigerj) [inline generated/python/gapic-google-cloud-logging-v2/setup.py]: I need to republish the package for this README to show up on PyPI. I cannot republish version 0.14.0; it is already published.

**Comment by ethanbao** [comment]
LGTM

**Comment by geigerj** [comment]
@bjwatson Could you take a look?

**Pull request merged**